main.go

```diff
@@ -3,14 +3,19 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/eldeal/skills/config"
+	"github.com/gorilla/mux"
 	"io/ioutil"
 	"log"
 	"net/http"
-
-	"github.com/gorilla/mux"
 )
 
 func main() {
+	// Get Config
+	cfg, err := config.Get()
+	if err != nil {
+		log.Println(fmt.Sprintf("error retrieving the configuration: [%s]", err.Error))
```

Comment:
We use our own [DP log library](https://github.com/ONSdigital/log.go) in all our apps to ensure that log output has consistent structure and can be parsed by our centralised logging tools.

Here is an example of the [import statement](https://github.com/ONSdigital/dp-dataset-api/blob/1b96e2e3001a41994f4395ecaeb0fe302810b2ba/api/dataset.go#L13) and using it logging an [error](https://github.com/ONSdigital/dp-dataset-api/blob/1b96e2e3001a41994f4395ecaeb0fe302810b2ba/api/dataset.go#L47)

Also because this is a "fatal error" you need to manually exit after logging the error
```
os.Exit(1)
```
Otherwise the app will continue and run in to further issues.

Hopefully thats helpful? Feel free to ask any questions if you have any 👍

Reply:
Thanks - @justinpjose had pointed out the DP log library but I was delaying using it 😅 . It's in now!

I'm not sure I used it correctly, particularly with regards to the context. Please let me know if it doesn't make sense